### PR TITLE
changed the caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @KalpanaReddyC
- [ ] @bommarajuphani7
- [ ] @rajkumarinfomagnus
- [ ] @aeruvakalpanaa

### PR DESCRIPTION
**Pull request by bommarajuphani7**
fixed #2

### CODE REVIEW

File: _slides/04-bommarajuphani7.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+training for develoers training
```
Comment (v-rajkumarinfomagnus) [inline _slides/04-bommarajuphani7.md]:
```suggestion
training for develoers training and awesom caption
```

File: _slides/04-bommarajuphani7.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![dinotocat](https://octodex.github.com/images/dinotocat.png)
```
Comment (v-rajkumarinfomagnus) [inline _slides/04-bommarajuphani7.md:8]: nice octacat

**Review by rajkumarinfomagnus — CHANGES_REQUESTED** [review]
make changes

File: _slides/04-bommarajuphani7.md (hunk 2 above)
Comment (bommarajuphani7) [reply _slides/04-bommarajuphani7.md:8]: thank you

**Review by rajkumarinfomagnus — APPROVED** [review]
ELTGM!!

**Pull request merged**